integration/querydsl/expressions/pom.xml

```diff
@@ -8,6 +8,8 @@
     </parent>
     <properties>
         <module.name>com.blazebit.persistence.querydsl</module.name>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Wasn't the whole project changed to Java 8 minimally already? I thought that was the blocker for `TREAT_INSTANT` and others to be added to begin with?

Reply:
It wasn't, the support of `java.time` types was doable without a hard dependency on Java 8. Version 2.0 which I'm working on currently will be based on Java 11 though, so we can get rid of some of the hacks.

Reply:
Ok, then we'll let this PR sit until 2.0 as well 😄